generate_workspace/src/main/java/com/google/devtools/build/workspace/output/BzlWriter.java

```diff
@@ -32,27 +35,33 @@
       MethodHandles.lookup().lookupClass().getName());
 
   private final String[] argv;
-  private final Path generatedFile;
+  private final File generatedFile;
```

Comment:
That's not an in-memory filesystem, it's just an on-disk filesystem that gets cleaned up.